neuralmonkey/runners/logits_runner.py

```diff
@@ -100,36 +83,25 @@ def __init__(self,
             raise ValueError("Either a pick index or a vocabulary value can "
                              "be specified, not both at the same time.")
 
-        self._pick_index = None  # type: Optional[int]
+        self.pick_index = None  # type: Optional[int]
```

Comment:
(není čas přejít na doopravické anotace z Pythonu 3.6?)

Reply:
Je, ale změní se tím 39 souborů.

Reply:
a uprostřed branche to udělat by byla sebemrskačská orgie